app/assets/stylesheets/accountants.scss

```diff
@@ -0,0 +1,9 @@
+// Place all the styles related to the dashboards controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+#search_form {
```

Comment:
Fine by me, I just hacked it in for the time being. The only other thing I can think of is making the views only load the CSS they need, but that's probably more hassle than it's worth (versus loading a global file). You could also just go full width on every class (even it the help text isn't that long).